docker/start.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Limit network traffic to 10kbit
+# tc qdisc add dev eth0 root tbf rate 1kbit latency 500ms burst 1024
+if [[ -z "${TEST_CMD}"  ]]; then
```

Comment:
Is there anything that sets TEST_CMD or is that maybe expected to be set by the user?